packages/main/src/components/AnalyticalTable/demo/generateData.ts

```diff
@@ -259,6 +259,7 @@ const makeTreeEntry = () => ({
 
 const makeEntry = () => ({
   name: getRandomName(),
+  longColumn: 'Really really long column content... don´t crop please',
```

Comment:
can you add the long column to the treeTableData as well? otherwise this column is always empty. Or change the columns for the treeTable back to the old config.